lib/compat/wordpress-6.8/class-gutenberg-hierarchical-sort.php

```diff
@@ -0,0 +1,201 @@
+<?php
+
+/**
+ * A custom REST server for Gutenberg.
```

Comment:
This comment is probably wrong

Reply:
https://github.com/WordPress/gutenberg/pull/66479/commits/c1961db37c8b1d62d985ae8fa435ec59458cd129